ebean-test/testconfig/ebean-db2-11.5.properties

```
@@ -0,0 +1,15 @@
+ebean.test.platform=db2
+ebean.test.dbName=unit
+# we need admin user to call the procedure SYSPROC.ADMIN_MOVE_TABLE
+ebean.test.username=admin
+ebean.test.password=admin
+datasource.default=db2
+ebean.db2.databasePlatformName=db2luw
+ebean.lengthCheck=utf8
+
+ebean.test.db2.version=11.5.9.0
+ebean.test.db2.containerName=ut_db2-11-5-9
+ebean.test.db2.port=50055
+ebean.test.db2.image=icr.io/db2_community/db2
```

this is wrong. See: https://github.com/ebean-orm/ebean-test-containers/pull/119
Either change to
```suggestion
ebean.test.db2.image=icr.io/db2_community/db2:11.5.9.0
```
or wait for fix in testcontainers and remove completely